content/en/Platform Deep Dive/Organization/automated-data-removal.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Automated Data Removal"
+linkTitle: "Automated Data Removal"
+weight: 40
+description: >
+  Process of automatically deleting data from the Cobalt Platform based on predefined rules and schedules.
+---
+
+{{% pageinfo %}}
+When your organization's contract expires, the associated data will be removed after a predefined period of time. This period can be extended based on certain conditions. Learn more about our [privacy policy](https://cobalt.io/terms#privacy).
+{{% /pageinfo %}}
+
+When your contract ends, a 15-day grace period begins during which you will still have access to the platform to [export any necessary data](/platform-deep-dive/organization/your-contract/#download-your-organizations-data). If the contract is renewed, full access to your account will be restored.
+
+{{% image src="/deepdive/ContractGateAfterGracePeriod.png" alt="contract gate after grace period" class="mt0" %}}
+
+After the grace period, you will still be able to sign in to the platform but will no longer have access to your organization's data. However, this data will remain in our system for 90 days after the contract ends.
+
+{{% image src="/deepdive/ConfirmToRetainData.png" alt="confirm to retain data" class="mt0" %}}
+
+You can elect to retain your account for an additional 90 days, if confirmed before the end of the initial period. The extension can only be performed once, and must be done by the Organization Owner(s) for your account.
+If you wish to have your data removed earlier, please reach out to your Customer Success Manager (CSM).
+
+## Data Subject to Removal
+
+The following data will be removed:
+
+- User accounts (except those associated with additional organizations not affected by data removal)
+- Organization's address, name, and logo
+- Assets
+- Pentests
```

Comment:
I would add vulnerabilities, because DAST and attack surface vulnerabilities are not linked to pentests. We could also add "other engagements" (DRA, SCR)